.sphinx/_templates/header.html

```diff
@@ -1,5 +1,26 @@
 <header id="header" class="p-navigation">
 
+  <script type="module" src="{{ pathto('_static/js/bundle.js', 1) }}">
+  </script>
+
+  <!-- Google Tag Manager -->
+  <script>
+    (function(w, d, s, l, i) {
+      w[l] = w[l] || [];
+      w[l].push({
+        'gtm.start': new Date().getTime(),
+        event: 'gtm.js'
+      });
+      var f = d.getElementsByTagName(s)[0],
+        j = d.createElement(s),
+        dl = l != 'dataLayer' ? '&l=' + l : '';
```

Comment:
This is quite a bit for a single code line.  I assume this is imported from elsewhere in which I understand desire to keep it as-is, however I think this would read cleaner broken up:

      var f = d.getElementsByTagName(s)[0];
      var j = d.createElement(s);
      var dl = l != 'dataLayer' ? '&l=' + l : '';

That last line is a bit confusing about what it's doing.  I might write it like this, just to make it clearer to debug:

    var dl = '';
    if (l != 'dataLayer') {
        dl = '&l=' + l;
    }

I'm not super guru in javascript but believe the performance impact of having it in a single line vs. four is going to be negligible.  Readability and maintainability usually trumps it.

Reply:
Implemented both your suggestions - I didn't notice any obvious performance hit from doing so, so sticking with that.